pep-0432.txt

```diff
@@ -51,9 +51,11 @@ well-defined phases during the initialization sequence:
 As a concrete use case to help guide any design changes, and to solve a known
 problem where the appropriate defaults for system utilities differ from those
 for running user scripts, this PEP also proposes the creation and
-distribution of a separate system Python (``pysystem``) executable
+distribution of a separate system Python (``system-python``) executable
 which, by default, operates in "isolated mode" (as selected by the CPython
-``-I`` switch).
+``-I`` switch), as well as the creation of an example stub binary that just
+runs an appended zip archive (permitting single-file pure Python executables)
```

Comment:
From what I recall, Thomas indicated that there were some problems with this approach which required sticking the archive into the elf data section.  It sounded like that was something different than just appending the zip archive.